lib/at_host/Kconfig

```diff
@@ -9,6 +9,7 @@ menu "AT Host Library for nrf91"
 config AT_HOST_LIBRARY
 	bool "AT Host Library for nrf91"
 	depends on BSD_LIBRARY
```

Comment:
I think the depends on can be removed because `AT_CMD` already depends on `BSD_LIBRARY`.

Reply:
ok